iamrahulmahato/master-web-development
# Added Odd-Even Game Against AI

### Pull Request Title:
Added Odd-Even Game Against AI | Closes #1108

### Description:
Implemented the Odd-Even Game Against AI using HTML, CSS, and JavaScript. The game allows users to select a number (0-9) and predict whether the sum of their number and AI’s random number will be odd or even.

### Changes Made:
- Developed the core game logic for player and AI interaction.
- Enhanced UI with:
  - Dark background.
  - Google Fonts (`Roboto`).
  - Spacing and larger fonts for labels and predictions.
- Default player number set to `0`.
- Improved output formatting with clear messages showing the game result.

### Test Instructions:
1. Select a number .
2. Predict odd or even.
3. Press **Play** and see if the prediction is correct.

![{CA9975E0-CF45-4E63-B6E1-E7D2B5ED09F5}](https://github.com/user-attachments/assets/79b2ea80-f96d-44bb-a78c-ed757b8d315c)


### Checklist:
- [x] Verified the game works as expected.
- [x] Added spacing and font size improvements.
- [x] Game logic matches the original issue's requirements.
